mono/mini/aot-runtime.c

```diff
@@ -5172,6 +5173,7 @@ mono_create_ftnptr_malloc (guint8 *code)
  *   Register a JIT icall which is called by trampolines in full-aot mode. This should
  * be called from mono_arch_init () during startup.
  */
+#undef mono_aot_register_jit_icall
```

Comment:
what is this about?

Reply:
You probably figured it out later. Macro form exists/existed to add a cast, i.e. from function pointer to gpointer which C silently allows, C++ does not.